### PR TITLE
added documentaion page

### PR DESCRIPTION
**Pull request by nerdyspook**
doc page

### CODE REVIEW

File: documentation/documentation.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                                class="nav__link"
+                                >Documentation</a
+                            >
+                        </li>
+                    </ul>
+                </div>
+                <div class="nav__btns">
+                    <a
+                        href="https://github.com/nerdyspook/Kal-UI-Component-Library"
+                        ><i class="fab fa-github" id="github-btn"></i>
```
Comment (ashishnnnnn) [inline documentation/documentation.html]: Just suggestion..
Can add target="_blank" for external links.. so that they open in new tab

**Review by ashishnnnnn — COMMENTED** [review]
Added one suggestion.. You can try that

File: documentation/documentation.html (hunk 1 above)
Comment (nerdyspook) [reply documentation/documentation.html]: Thanks for the suggestion, I will do that for all the links which need this.

**Review by devananth — COMMENTED** [review]
Code looks clean. You can merge it

**Pull request merged**